conf.d/arkmanager-user.cfg

```diff
@@ -16,7 +16,7 @@ ark_ServerPassword=${SERVER_PASSWORD}                               # ARK server
 ark_ServerAdminPassword=${ADMIN_PASSWORD}                           # ARK server admin password, KEEP IT SAFE!
 ark_MaxPlayers=${MAX_PLAYERS}                                       # Number MAX of player
 ark_GameModIds=${GAME_MOD_IDS}                                      # Uncomment to specify additional mods by Mod Id separated by commas
-#ark_AltSaveDirectoryName="SotF"                                    # Uncomment to specify a different save directory name
+ark_AltSaveDirectoryName="${ALTSAVEDIRNAME:-SavedArks}"             # Uncomment to specify a different save directory name
```

Comment:
Did you verify that Parameter-Expansion works here?  I'm not sure because this is a bash behaviour